README.md

```diff
@@ -1,4 +1,4 @@
-# [project name]
+# Leaderboard
 
 ## Built With
 
```

Comment:
Please fix the way you're linking your LinkedIn profile in the `Author` section below. Make sure you wrap the content you want to link in [] and the actual link in ().